app/controllers/api/users.php

```diff
@@ -1211,6 +1372,44 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $response->noContent();
     });
 
+App::delete('/v1/users/:userId/targets/:targetId')
+    ->desc('Delete user target')
+    ->groups(['api', 'users'])
+    ->label('audits.event', 'users.targets.delete')
+    ->label('audits.resource', 'target/{request.$targetId}')
+    ->label('scope', 'targets.write')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'users')
+    ->label('sdk.method', 'deleteTarget')
+    ->label('sdk.description', '/docs/references/users/delete-target.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_NONE)
+    ->param('userId', '', new UID(), 'User ID.')
+    ->param('targetId', '', new UID(), 'Target ID.')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->inject('events')
+    ->action(function (string $userId, string $targetId, Response $response, Database $dbForProject, Event $events) {
+
+        $user = $dbForProject->getDocument('users', $userId);
+
+        if ($user->isEmpty()) {
+            throw new Exception(Exception::USER_NOT_FOUND);
+        }
+
+        $target = $dbForProject->getDocument('targets', $targetId);
+
+        if ($target->isEmpty()) {
+            throw new Exception(Exception::USER_TARGET_NOT_FOUND);
+        }
+
+        $target = $dbForProject->deleteDocument('targets', $target->getId());
```

Comment:
Let's check the target belongs to the user before deleting

Reply:
done this